src/legacy/upload/upload.stories.js

```diff
@@ -0,0 +1,32 @@
+import { html } from 'lit-html';
+import '../upload';
+import styles from '../../components/button/styles.scss';
+import { stylesheet, docsIntro } from '../../../.storybook/utils.js';
+import linkStyles from '../../components/link/styles.scss';
+
+export default {
+  title: 'legacy/vl-upload',
+  decorators: [(story) => html`${stylesheet(`${styles}${linkStyles}`)}${story()}`],
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'upload',
+          intro:
```

Comment:
Zouden we die intro wel toevoegen? Gewoon doorverwijzen naar de docs lijkt me voldoende.